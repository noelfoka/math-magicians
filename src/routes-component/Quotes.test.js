import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quotes'; // Adjust the import path

describe('Quote component', () => {
  it('should match QUOTE snapshot', () => {
    const quote = renderer.create(<Quote />).toJSON();
    expect(quote).toMatchSnapshot();
  });
});
