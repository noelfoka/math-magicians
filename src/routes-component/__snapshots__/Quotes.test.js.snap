// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component should match QUOTE snapshot 1`] = `
<div
  className="quotes"
>
  <h1>
    I love this very motivational statement:
  </h1>
  <hr />
  <p>
    Loading...
  </p>
</div>
`;
